project/auth/routes.py

```diff
@@ -0,0 +1,189 @@
+# Modules from Flask Library
+from flask import Flask, render_template, request, flash, Blueprint
+from flask import session as login_session
+from flask import make_response, redirect, jsonify, url_for
+# SQLAlchemy library to access database
+from sqlalchemy import create_engine, asc
+from sqlalchemy.orm import sessionmaker
+from project.models import Base, Category, Product, User
+
+import random
+import string
+import httplib2
+import json
+import requests
+
+# Authentification modules
+from oauth2client.client import flow_from_clientsecrets
+from oauth2client.client import FlowExchangeError
+
+
+auth = Blueprint('auth', __name__)
+
+# Connect to Database and create database session
+engine = create_engine('sqlite:///furniturecatalog.db',
+                       connect_args={'check_same_thread': False})
+# Bind the engine to the metadata of the Base class so that the
+# declaratives can be accessed through a DBSession instance
+Base.metadata.bind = engine
+
+DBSession = sessionmaker(bind=engine)
+# A DBSession() instance establishes all conversations with the database
+# and represents a "staging zone" for all the objects loaded into the
+# database session object. Any change made against the objects in the
+# session won't be persisted into the database until you call
+# session.commit(). If you're not happy about the changes, you can
+# revert all of them back to the last commit by calling
+# session.rollback()
+session = DBSession()
+
+
+CLIENT_ID = json.loads(
+    open('client_secrets.json', 'r').read())['web']['client_id']
```

Comment:
I would suggest creating a config service so you could do something like this:
```suggestion
config_service = ConfigService()    
CLIENT_ID = config_service.get_setting('web_client_id')
```

`ConfigService` would just load and parse JSON on init and store it in a class variable.